addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/HiddenFilesScanRule.java

```diff
@@ -72,6 +72,8 @@ public class HiddenFilesScanRule extends AbstractHostPlugin {
                     CommonAlertTag.OWASP_2017_A06_SEC_MISCONFIG,
                     CommonAlertTag.WSTG_V42_CONF_05_ENUMERATE_INFRASTRUCTURE);
     static final String PAYLOADS_FILE_PATH = "json/hidden_files.json";
+    static final String DEFAULT_PAYLOAD_PATH =
+            Constant.getZapHome() + File.separator + PAYLOADS_FILE_PATH;
```

Comment:
Deploying the file in the tests would require less changes to the scan rule, but fine as is. I'd remove the extra validation added to `readPayloadsFile` though.